muspinsim/muspinsim.xml

```diff
@@ -12,7 +12,7 @@
                 license = {MIT},
```

Comment:
You're also an author on v1.1.0 so you need to be added to the citation, but I can't make a suggestion for the author line, for some reason.